packages/pagination/src/Pagination.js

```diff
@@ -9,14 +9,15 @@ export const PaginationContext = React.createContext();
 export const usePagination = () => useContext(PaginationContext);
 
 const getItemLocalStorage = (key) => {
```

Comment:
Having made this change, if you look on lines [73](https://github.com/Availity/availity-react/pull/1035/files#diff-10242440ac53333e129b7a0a67ab4d395ef1534719be90d98d37384d712a5c35R73) and then [96](https://github.com/Availity/availity-react/pull/1035/files#diff-10242440ac53333e129b7a0a67ab4d395ef1534719be90d98d37384d712a5c35R96) the check seems fairly useless. Feels like we're setting 'current-page' to the value of `currentPage` and then checking to see if 'current-page' equals `currentPage`. Am I missing something?

Reply:
I think you're right. It should be equal every time in this scenario. Might be a relic from the past? I think it would be safe to remove